### ISSUE TITLE
types-aiobotocore README.md is broken

### ISSUE THREAD

**Issue by ytoku** [youtype/mypy_boto3_builder#161]
**Describe the bug**
README.md of types-aiobotocore-* has invalid indent at paginators and waiters code blocks.
Especially, invalid indent of ```` ``` ```` breaks following markups.

**To Reproduce**
Generate REDME.md of types-aiobotocore-*.

**Actual output**

For example, types-aiobotocore-s3 Paginators annotations section is

```
    # Explicit type annotations are optional here
    # Type should be correctly discovered by mypy and IDEs
    # VSCode requires explicit type annotations
        list_multipart_uploads_paginator: ListMultipartUploadsPaginator = client.get_paginator("list_multipart_uploads")
        list_object_versions_paginator: ListObjectVersionsPaginator = client.get_paginator("list_object_versions")
        list_objects_paginator: ListObjectsPaginator = client.get_paginator("list_objects")
        list_objects_v2_paginator: ListObjectsV2Paginator = client.get_paginator("list_objects_v2")
        list_parts_paginator: ListPartsPaginator = client.get_paginator("list_parts")
    ```
```

**Expected output**
It should be

````
    # Explicit type annotations are optional here
    # Type should be correctly discovered by mypy and IDEs
    # VSCode requires explicit type annotations
    list_multipart_uploads_paginator: ListMultipartUploadsPaginator = client.get_paginator("list_multipart_uploads")
    list_object_versions_paginator: ListObjectVersionsPaginator = client.get_paginator("list_object_versions")
    list_objects_paginator: ListObjectsPaginator = client.get_paginator("list_objects")
    list_objects_v2_paginator: ListObjectsV2Paginator = client.get_paginator("list_objects_v2")
    list_parts_paginator: ListPartsPaginator = client.get_paginator("list_parts")
```
````

**Comment by vemel** [comment]
Thank you for the report! It is sometimes hard to preserve proper indentation in `jinja2` templates. However, should be easy to fix.

**Comment by vemel** [comment]
I have released `types-aiobotocore 2.3.4.post3` with the fix included. Also, all code blocks in `README` files are formatted with `black`. Please check and let me know if it looks as it should.

https://pypi.org/project/types-aiobotocore-s3/#paginators-annotations

**Comment by ytoku** [comment]
I have confirmed the indentation is fixed. Thanks!
